lib/azure.js

```diff
@@ -1355,6 +1355,67 @@ exports.createInsightsClient = function (credentials, subscriptionId, options) {
   return new AzureInsights(credentials, subscriptionId, null, options);
 };
 
+/**
+ * @class
+ * Initializes a new instance of the MonitorManagementClient class.
+ * @constructor
```

Comment:
avoid the `@class` and `@constructor` syntax. This breaks docs.microsoft.com as they parse our source code. See other usages in this file for reference or see [jsdoc](usejsdoc.org/howto-es2015-classes.html).